sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -1009,6 +1041,90 @@ public async Task GetBatchSettingIfChangedWithUnmodifiedPageSync()
             Assert.AreEqual(2, pagesCount);
         }
 
+        [RecordedTest]
+        [AsyncOnly]
```

Comment:
We need two separate tests, both async and sync, because we're skipping client instrumentation to make some pageable tests work due to a Test Framework limitation.